components/global/product_branding/product_branding.tsx

```diff
@@ -11,6 +11,7 @@ import {useProducts, useCurrentProductId} from '../hooks';
 const ProductBrandingContainer = styled.div`
     display: flex;
     align-items: center;
+    cursor: pointer;
```

Comment:
I'm wondering if this should maybe be relocated to `ProductSwitcherContainer` since the `onClick` handler is there so we can keep the functionality and the visual affordance for the functionality "together".